sdk/python/tests.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
+PYTHONPATH=. SCHEMAS_PATH="../../catbuffer/schemas/test-vectors" pytest tests/generator
```

Comment:
shouldn't this go in vectors.sh, not tests.sh ?

Reply:
this is _much_ faster than running test-vectors... so imo can go here
earlier both catbuffer test vectors and (big) test vectors were part of a common stage `test`, but catbuffer tests did not have branch limitation:

```yaml
    - stage: test
      name: unit tests
# ...
    - name: catbuffer test vectors
# ...
    - name: test vectors
      if: branch = env(DEV_BRANCH) OR branch = env(RELEASE_BRANCH)
```

Reply:
these are vectors though, don't really want to run through them when running unit tests 😅

Reply:
tomato, tomato ^^
`unittest discover` - `384 tests in 1.943s`
catbuffer "vectors" - `188 passed in 0.89s`